container/karaf/src/main/filtered-resources/etc/custom.properties

```diff
@@ -710,6 +710,7 @@ org.osgi.framework.system.packages.extra=org.apache.karaf.branding,\
         org.opennms.core.rpc.api;version=${opennms.osgi.version},\
         org.opennms.core.rpc.echo;version=${opennms.osgi.version},\
         org.opennms.core.utils.url;version=${opennms.osgi.version},\
+        org.opennms.features.activemq.broker.api;version=${opennms.osgi.version},\
```

Comment:
Here we do change a configuration file, but users shouldn't normally be changing this anyhow.